components/dydx_order_manager.py

```diff
@@ -277,6 +278,112 @@ def withdraw_from_dydx(self, recipient_address, dydx_client):
+        asset_open_position = dydx_order_manager.position_status(
+            "position_status", order_details["binance_asset_pair"]
+        )
+        data = dydx_order_manager.get_asset_price_and_size(
+            order_details["asset_oracle_address"], dydx_asset_instance
+        )
+        asset_market_price = data["price"]
+
+        asset_position_size = data["size"]
+
+        if order_details["asset_trigger_price"] is None:
+            print("Start::calculate open close")
+            asset_trigger_price = dydx_order_manager.calculate_open_close_price(
+                asset_pair=order_details["asset_pair"],
+                eth_order_size=float(asset_position_size),
+                symbol=order_details["binance_asset_pair"],
+            )
+        else:
+            asset_trigger_price = asset_market_price + 10
+        if asset_open_position is False:
+
+            if asset_trigger_price >= asset_market_price:
+                order_params = dydx_order_manager.create_order_params(
+                    order_details["order_side"],
+                    order_details["asset_pair"],
+                    asset_position_size,
+                    asset_market_price,
+                    dydx_asset_instance,
+                )
+                dydx_order_manager.create_order(order_params, dydx_asset_instance)
+                asset_open_position = True
+                dydx_order_manager.set_position_status(
+                    collection_name="position_status",
+                    symbol=order_details["binance_asset_pair"],
+                    status=asset_open_position,
+                )
+                print(f"{order_details['asset_pair']} - Short position is open")
+            else:
+                print(
+                    f"{order_details['asset_pair']} {asset_trigger_price} is less than market price {asset_market_price}"
+                )
+        else:
+            if asset_trigger_price >= asset_market_price:
+                print(f"{order_details['asset_pair']} position is already open")
+            else:
+                print(
+                    f"{order_details['asset_pair']} trigger price {asset_trigger_price} is less than market price {asset_market_price}"
+                )
+
+    def close_order_on_dydx(self, order_details):
+        dydx_asset_instance = asset_dydx_instance[order_details["asset_pair"]]
+        dydx_order_manager = DydxOrderManager(dydx_asset_instance)
+        asset_open_position = dydx_order_manager.position_status(
+            "position_status", order_details["binance_asset_pair"]
+        )
+        data = dydx_order_manager.get_asset_price_and_size(
+            order_details["asset_oracle_address"], dydx_asset_instance
+        )
+        asset_market_price = data["price"]
+
+        asset_position_size = data["size"]
+
+        if order_details["asset_trigger_price"] is None:
+            print("Start::calculate open close")
```

Comment:
is this print statement needed ? 
If yes, structure the print statement as such:
For celery tasks, "Start :: <name of the task>"
For any other file, "ClassName :: <print statement>"

Reply:
Do make the necessary changes for every print statement

Reply:
sure

Reply:
oh I think there are some print statements inside dydx_order_manager functions without the above convention